### ISSUE TITLE
When delegating, how to know who is who to delegate to

### ISSUE THREAD

**Issue by mitar** [peer/mind#177]
So the idea of delegation is that you delegate to people you know and share similar values. If this is something you know from the behavior on the PeerMind itself, then it is easy to delegate to their username. But if this is somebody you know in real life, then the question is how you find them on PeerMind. What is username is cryptic? If the do not disclose their name? Or if avatar is not a photo? And on the other hand, what if somebody puts a name and username which looks like somebody else? Or photo? And trick users to delegate to them, instead of the real person?

So in some way it is not enough to know that all people in the system are real people of the community and that they have one vote per person (so no Sybil attack). It is also important that each of those individuals delegating to others can verify in an easy but secure way that they are really delegating to those they believe they are delegating to.

This might be addressed by a community having fixed rules on usernames and names (real name policy or something), maybe it could be addressed by having ways to make "verified" mark or something (like with Facebook and Twitter and other links; or even Keybase). That the name user by the user matches the one on Facebook, Twitter, Keybase.

**Comment by subjective** [comment]
What if there is a feature that allows a user to reveal/prove their Facebook/Twitter/Keybase identities to selected users, without necessarily revealing those identities to anyone else?  If I want to prove my identity to another user, I can perform an explicit action in the app to do so.

There could be ways of making this easier, e.g. to have a setting that automatically reveals my FB identity only to my friends.  In this case, other users can see my identity and "verified" mark if they connected their FB accounts and we are friends.  There could be similar settings for the users I follow on Twitter or Keybase.

So, basically, when you link your FB/Twitter/Keybase account, you have three settings:

Who can see this identity by default?
* Everyone
* Only friends or users I follow
* Nobody

In the latter two options, you can still manually reveal your identity to anyone.

**Comment by mitar** [comment]
>  If I want to prove my identity to another user, I can perform an explicit action in the app to do so.

There is one one issue if we do not want delegates to know that somebody is delegating to them. Then in some way any communication (like asking for proof) might reveal the fact that I am interested in delegating to you.

> There could be ways of making this easier, e.g. to have a setting that automatically reveals my FB identity only to my friends. In this case, other users can see my identity and "verified" mark if they connected their FB accounts and we are friends. There could be similar settings for the users I follow on Twitter or Keybase.

I think this is a great idea! In general I was struggling to how to expose the information about linked accounts. I think this would be a great default in general.

We can also comment below that revealing more information can help others determine the correct person to delegate to.

We could also make a filter on the list of users to delegate to for "all my friends from facebook" and similar.

**Comment by mitar** [comment]
I opened #180 for the option to reveal linked information through these options.

Still, are there any other ways?
